x-element.js

```diff
@@ -21,6 +22,24 @@ export default class XElement extends HTMLElement {
     return XElement.defaultTemplateEngine;
   }
 
+  /**
+   * Declare an initialization object for the host’s shadow root.
+   * See https://developer.mozilla.org/en-US/docs/Web/API/Element/attachShadow.
+   */
+  static get shadowRootInit() {
+    return { mode: 'open' };
+  }
+
+  /**
+   * Declare an array of CSSSTyleSheet objects to adopt on the shadow root.
+   * Note that a CSSStyleSheet object is the type returned when importing a
+   * stylesheet file via import attributes. This has no effect if you are using
+   * the “host” as your render root (versus attaching a shadow root).
+   */
+  static get styleSheets() {
```

Comment:
This is the crux of the change. We can fuss over the name here. I personally like `cssStyleSheets`, `styleSheets`, or `styles` as the options. But, I’m willing to be convinced otherwise 😸

Reply:
```
import MyStylesAreTheBest from './path-to.css' with { type: 'css' };
import YourStyles from 'https://your.style/sheet.css' with { type: 'css' };

static get styles() {
  
  const rules = `:host { display: block }`;
  const inlineStyles = new CSSStyleSheet();
  inlineStyles.replaceSync(rules);

  return [
    MyStylesAreTheBest,
    YourStyles,
    inlineStyles,
  ];
}
```